src/nvim/charset.c

```diff
@@ -1240,7 +1250,7 @@ void getvcol(win_T *wp, pos_T *pos, colnr_T *start, colnr_T *cursor,
           if (enc_utf8 && (c >= 0x80)) {
             incr = utf_ptr2cells(ptr);
           } else {
-            incr = CHARSIZE(c);
+            incr = char_size(c);
```

Comment:
Wondering how you managed to pass tests with this problem. I am pretty sure that passing int to uint8_t should file -Wconversion warning, even with clang. Does this mean that charset.c is not checked? I see no exceptions made for any file.